sale_order_line_price_history/views/sale_views.xml

```diff
@@ -9,15 +9,18 @@
         <field name="arch" type="xml">
             <xpath expr="//field[@name='order_line']/tree">
                 <field name="order_partner_id" invisible="1" />
-                <widget
-                    name="sale_line_price_history_widget"
-                    width="20px"
+                <field
+                    name="id"
+                    string="Price Widget"
+                    context="{'active_id': id, 'active_ids': [id]}"
                     attrs="{'invisible': [('id', '!=', False)]}"
+                    widget="sale_line_price_history_widget"
```

Comment:
There's not too much sense to have this column, and more having the button next to it. Right now is taking a lot of space for nothing and confusing user.

Reply:
Yeah, the problem is that the widget is not doing its work. Talked by chat, so you know what to do.